assets/onboarding/welcome.jsx

```diff
@@ -0,0 +1,61 @@
+import { Card, H, Link } from '@woocommerce/components';
```

Comment:
I was using one component (and support components) per folder structure - in this case, I'd create `welcome/index.jsx`. I like it because if we have some specific style file, test file, stories, or other things related, we can add it together. And I think it's a good structure to grow organized. What do you think? If you have a different opinion, we can discuss and I can refactor where I used this structure.

Reply:
Good approach, I moved them in bb6d287.